src/java.desktop/macosx/native/libawt_lwawt/awt/JavaComponentAccessibility.m

```diff
@@ -1883,7 +1876,7 @@ - (jobject)tabGroup
 @end
 
 
-@implementation ScrollAreaAccessibility
+@implementation ScrollAreaAccessibility_Old
```

Comment:
It will not be instantiated ever with these changes. I was not very sure whether to keep it or not, that is why I mentioned it in the description itself. I was thinking may want to fall back to old API sometime to test if everything works fine after the changes. So I decided to keep it. But even I feel it messes up the code and we can always revert locally if we want to test. So removing the old interface and implementation.